proto/viam/app/slammapping/v1/slammapping.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package viam.app.slammapping.v1;
+
+import "app/data/v1/data.proto";
+import "common/v1/common.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/slammapping/v1";
+
+service CloudSLAMService {
+    rpc StartMappingSession(StartMappingSessionRequest) returns (StartMappingSessionResponse);
+    rpc GetActiveMappingSessionsForRobot(GetActiveMappingSessionsForRobotRequest) returns (GetActiveMappingSessionsForRobotResponse);
+    rpc GetMappingSessionPointCloud(GetMappingSessionPointCloudRequest) returns (GetMappingSessionPointCloudResponse);
+    rpc ListMappingSessions(ListMappingSessionsRequest) returns (ListMappingSessionsResponse);
+    rpc StopMappingSession(StopMappingSessionRequest) returns (StopMappingSessionResponse);
+}
+
+// StartMappingSession
+
+message StartMappingSessionRequest {
+    // the config required to run slam. This includes a replay sensor and map package  
+    google.protobuf.Struct slam_config = 1; //for cartographer/algo specific fields
+    string slam_version = 2; // version to use for slam, defaults stable
+    string map_name = 3;
+}  
+
+message StartMappingSessionResponse {
+    string session_id = 1;
+}
+  
+// GetMappingSession
+
+message GetMappingSessionRequest {
+    string id = 1;
+}
+
+message ReplaySensor{
+    string org_id = 1;
+    string location_id = 2;
+    string robot_id = 3;
+    // source component that the replay sensor will use
+    string source = 4;
+    viam.app.data.v1.CaptureInterval interval = 5;
+}
```

Comment:
I think this is actually no longer needed, since we decided the StartMappingSessionRequest's slam_config will contain the replay sensor. I'm not 100% sure since @JohnN193  left it in the spec, but since it's currently unused, could we leave it out of this PR? I added a comment on the scope.

Reply:
Thanks, removed! Will ask when he's back